### PR TITLE
feat(ui): Custom  'Hello Microverse' Header Text 

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Add Hello Microverse Text
In this feature update:
- Added Index.html file
- Added index.css file
- Linked index.css to index.html file
- Fixed HTML CSS errors raised by linters

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.header-title {
```
Comment (carlos-ssh) [inline index.css]: - Kindly fix your linter issues displayed on your Pull Request.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @blessedjasonmwanza , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 1 above)
Comment (blessedjasonmwanza) [reply index.css]: Hello @carlos-ssh,

Thank you for the review.

I have run linter tests locally and I see no errors raised. I am currently running node 14.17.6 and post taking a read on some similar solutions to the error, online resources are advising that I delete the node modules and package.lock.json.

What would you advise that I do in this scenario?

Comment (carlos-ssh) [reply index.css]: It's needed to check locally and in your pull request which error do you have.  you can go to your linters described in the next image and get a detail about your issue. Solve it and after that request another code review.

Screen Shot 2021-09-14 at 7 56 00

Comment (blessedjasonmwanza) [reply index.css]: Hello, once more @carlos-ssh,

Thank you a lot for that insightful suggestion.

I have taken some time to look at the error raised and it seems it's coming from a missing dependency program.

Kindly see the attached herewith image for your reference. Meanwhile, I will go ahead and make another pull request as you advised earlier.

stylelint error

Comment (blessedjasonmwanza) [reply index.css]: Hi @carlos-ssh,

>I believe I am done following your advice.

I am requesting another review, and hopefully this time I check out fine 😃

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello @blessedjasonmwanza  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**